Sources/SwiftSyntaxBuilderGeneration/Templates/BuildableCollectionNodesFile.swift

```diff
@@ -200,7 +200,7 @@ private func createBuildFunction(node: Node) -> FunctionDecl {
                         BinaryOperatorExpr("+")
                         FunctionCallExpr(MemberAccessExpr(base: "format", name: "_makeIndent"))
                       }
-                    : "nil"
+                    : NilLiteralExpr()
```

Comment:
Should it be just a keyword? 
```suggestion
                    : SyntaxFactory.makeNilKeyword(trailingTrivia: [])
```

I'd like to get rid of default trailing space after https://github.com/apple/swift/pull/60204 is merged

Reply:
Not sure, a token syntax can be converted [to identifiers and binary operators](https://github.com/apple/swift-syntax/blob/2380374d1ed52d89674adad21d98b8dd1d1be14c/Sources/SwiftSyntaxBuilder/gyb_helpers/ExpressibleAsConformances.py#L41), neither of which seem to be correct in this case.

Since the expression is a nil literal, `NilLiteralExpr` should be the right way to spell it in `SwiftSyntaxBuilder` too IMO